frontend/web/components/modals/payment/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const ENTERPRISE_ICON_COLOR = '#F7D56E'
```

Comment:
Ah yes, we already have constants (see comment on the pricingFeatures), maybe it would be noisy to have them there but an indication that the other file is declaration focused would help